content/docs/started/workstation/getting-started-linux.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Kubeflow on Linux"
+description = "Install Kubeflow on Linux"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+  more quickly
+-->
+
+## Introduction
+
+For Linux systems you have options for servers and desktops. The server options apply to
+the desktop as well.
+
+## Linux Server
```

Comment:
Change all upper case to lower case where possible, here, and in all headings on this page, and in all files. Our docs use the pattern of title case for page titles, and sentence case everywhere else. Brand names get upper case (or whatever case is dictated by the brand).

The [Kubeflow style guide](https://www.kubeflow.org/docs/about/style-guide/) has useful info about this and other hints for improving clarity of the docs.